lib/puppet/parser/resource.rb

```diff
@@ -181,15 +181,21 @@ def to_ral
     copy_as_resource.to_ral
   end
 
-  # Is the receiver tagged with the given tags?
+  # Faster variant of the tagged method that does no conversion of its
```

Comment:
We want the first line of the comment to be on the form "Returns...", "Answers...", "Computes...", "Adds, "... etc.

Suggest saying "Answers if this resource is tagged with one of the tags given in downcased string form.", and then adjust the rest of the documentation of this method accordingly.